class-wc-calypso-bridge.php

```diff
@@ -108,6 +115,16 @@ public function __construct() {
 				'wc-admin-selling-online-courses', // hide for now, to be revisited.
 			);
 
+			// Do not show the message if the site is active for less than 5 days.
+			if ( ! WCAdminHelper::is_wc_admin_active_for( 5 * DAY_IN_SECONDS ) ) {
```

Comment:
The Note will be added to active stores for at least five days. Right? Why is there no need to check for the `W44_2022_RELEASE_DATE` value somewhere?  The same goes for the other one below.

Reply:
Thank you @xristos3490 - There is no need to check for `W44_2022_RELEASE_DATE` as this message, when it gets added is added with the check for `W44_2022_RELEASE_DATE`

After our call, I renamed the variable, to make more sense; since we have negative logic, it will be clearer for someone studying the code